content/docs/rosa/alb-nlb-ossm-e2e/index.md

```diff
@@ -0,0 +1,530 @@
+---
+date: '2023-07-24'
+title: Securely Exposing Applications with PrivateLink ROSA Cluster and OpenShift Service Mesh
+tags: ["ROSA", AWS, "OpenShift Service Mesh,Private Link"]
+authors:
+  - Mohsen Houshamnd
+  - Paul Czarkowski
+---
+
+This repository demonstrates how to utilize a privatelink ROSA (Red Hat OpenShift on AWS) cluster and OSSM (OpenShift Service Mesh) to securely expose an application with end-to-end encryption. The provided architecture serves as a sample for application exposure. The deployment incorporates an Ingress/Egress VPC to route traffic to the cluster, which is deployed within a private VPC.
+
+![architecture diagram showing privatelink with TGW](./images/rosa_alb_tgw_nlb_e2e.png)
+
+### Prerequisites
+
+- A privatelink ROSA cluster
+- [ROSA CLI](https://github.com/openshift/rosa) - Download the latest release
+- oc CLI `rosa download openshift-client`
+- [jq](https://jqlang.github.io/jq/download/)
+
+Clone the repository
+
+```bash
+git clone https://github.com/houshym/rosa-ossm-e2e-encryption
+```
+ ### Deploy OpenShift Service Mesh(OSSM)
+
+ **Note:** if service mesh is already installed skip this step
+
+Installing the OSSM(OpenShift Service Mesh) involves installing the OpenShift Elasticsearch, Jaeger, Kiali, and Service Mesh Operators, creating and managing a ServiceMeshControlPlane resource to deploy the control plane, and creating a ServiceMeshMemberRoll resource to specify the namespaces associated with the Service Mesh 
+
+Install service mesh operators by applying the following snippet on the cluster or use this [script](./ossm-operator/deploy-ossm.sh)
```

Comment:
the script should not be a hyperlink but an instruction "or run the script inside the cloned git repo `./ossm-operator/deploy-ossm.sh`  ...  "

Reply:
done